rita/src/rita_common/port_emissary/mod.rs

```diff
@@ -0,0 +1,45 @@
+use std::fs::File;
```

Comment:
I'd argue this goes as a Kernel interface module, as we (try to) set it up as a pluggable interface for different operating systems we may need to work with.